src/Explorer/Panes/UploadItemsPane/index.tsx

```diff
@@ -0,0 +1,220 @@
+import { DetailsList, DetailsListLayoutMode, IColumn, SelectionMode } from "office-ui-fabric-react";
+import React, { ChangeEvent, FunctionComponent, KeyboardEvent, useEffect, useRef, useState } from "react";
+import FolderIcon from "../../../../images/folder_16x16.svg";
+import * as Constants from "../../../Common/Constants";
+import { userContext } from "../../../UserContext";
+import { logConsoleError } from "../../../Utils/NotificationConsoleUtils";
+import { UploadDetails, UploadDetailsRecord } from "../../../workers/upload/definitions";
+import Explorer from "../../Explorer";
+import { getErrorMessage } from "../../Tables/Utilities";
+import { GenericRightPaneComponent, GenericRightPaneProps } from "../GenericRightPaneComponent";
+import { Tooltip } from "./Tooltip";
+
+const UPLOAD_FILE_SIZE_LIMIT = 2097152;
+
+export interface UploadItemsPaneProps {
+  explorer: Explorer;
+  closePanel: () => void;
+}
+
+interface IUploadFileData {
+  numSucceeded: number;
+  numFailed: number;
+  fileName: string;
+}
+
+const getTitle = (): string => {
+  if (userContext.apiType === "Cassandra" || userContext.apiType === "Tables") {
+    return "Upload Tables";
+  } else if (userContext.apiType === "Gremlin") {
+    return "Upload Graph";
+  } else {
+    return "Upload Items";
+  }
+};
+
+export const UploadItemsPane: FunctionComponent<UploadItemsPaneProps> = ({
```

Comment:
Is it possible to paste a snapshot of the end result in this PR, similar to #579 ? Thanks!

Reply:
done